Neos.Flow/Resources/Private/Schema/Settings/Neos.Flow.security.schema.yaml

```
@@ -74,6 +74,8 @@ properties:
     additionalProperties: false
     required: true
     properties:
+      'encryptionKey': { type: string, required: true }
```

```suggestion
      'encryptionKey': { type: [string, 'null'], required: true }
```

I guess the schema sould look like this. (untested)